buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/IntellijVersions.kt

```diff
@@ -1,12 +1,10 @@
 package com.virtuslab.gitmachete.buildsrc
 
 import com.virtuslab.gitmachete.buildsrc.IntellijVersionHelper.getPropertyOrNullIfEmpty
+import java.util.Properties
 
 // See https://www.jetbrains.com/intellij-repository/releases/ -> Ctrl+F .idea
-object IntellijVersions {
-
-  private val intellijVersionsProp = IntellijVersionHelper.getProperties()
-
+public class IntellijVersions(overrideBuildTarget: String?, intellijVersionsProperties: Properties) {
```

Comment:
Nit: I'd invert the order of these two params, `intellijVersionsProperties` is much more important of the two